benchmarks/linear-elastic-plate-with-hole/workflow_creation_guide.md

```diff
@@ -0,0 +1,930 @@
+├── nextflow.config                 # Nextflow configuration
+│
+└── tool1/                          # Directory for simulation tool 1
+    ├── Snakefile                   # Tool-specific Snakemake workflow
+    ├── tool1.nf                    # Tool-specific Nextflow workflow
+    ├── environment_simulation.yml  # Tool-specific conda environment
+    ├── run_tool1_simulation.py     # Main simulation script
+    ├── create_tool1_input.py       # Input file preparation (if needed)
+    ├── postprocess_results.py      # Post-processing script (if needed)
+    └── template_files/             # Template configuration files
+```
+
+## Algorithm for Creating Dual-System Workflows
+
+### Step 1: Define Parameter Files and Configuration
+
+Create parameter files following the standardized JSON format:
+
+```json
+{
+    "configuration": "unique_config_name",
+    "geometry_parameters": {
+        "radius": {"value": 0.33, "unit": "m"},
+        "length": {"value": 1.0, "unit": "m"}
+    },
+    "material_properties": {
+        "young-modulus": {"value": 210e9, "unit": "Pa"},
+        "poisson-ratio": {"value": 0.3, "unit": ""}
+    },
+    "mesh_parameters": {
+        "element-size": {"value": 0.1, "unit": "m"},
+        "element-order": 1,
+        "element-degree": 1
+    },
+    "solver_settings": {
+        "quadrature-rule": "gauss",
+        "quadrature-degree": 1
+    },
+    "loading_conditions": {
+        "load": {"value": 100.0, "unit": "MPa"}
+    }
+}
+```
+
+### Step 2: Implement Configuration Generator
+
+Create `generate_config.py` to automatically discover and validate configurations:
+
+```python
+import json
+from pathlib import Path
+
+def generate_workflow_config():
+    # Discover parameter files
+    files = list(Path(".").glob("parameters_*.json"))
+    
+    # Extract and validate configurations
+    configurations = {}
+    for file in files:
+        with open(file, 'r') as f:
+            data = json.load(f)
+        config_name = data.get("configuration", file.stem.split("_")[1])
+        configurations[file] = config_name
+    
+    # Check for duplicates
+    config_values = list(configurations.values())
+    duplicates = set([x for x in config_values if config_values.count(x) > 1])
+    if duplicates:
+        raise ValueError(f"Duplicate configurations: {duplicates}")
+    
+    # Create workflow configuration
+    workflow_config = {
+        "configuration_to_parameter_file": {v: str(k) for k, v in configurations.items()},
+        "configurations": list(configurations.values()),
+        "tools": ["fenics", "kratos"],  # Update with your tools
+        "benchmark": "your-benchmark-name"
+    }
+    
+    # Save configuration
+    with open("workflow_config.json", "w") as f:
+        json.dump(workflow_config, f, indent=4)
+    
+    return workflow_config
+
+if __name__ == "__main__":
+    generate_workflow_config()
+```
+
+### Step 3: Implement Mesh Generation
+
+Create `create_mesh.py` with standardized interface:
```

Comment:
see above (just link), but also discuss the option of providing all meshes (or a single one if that only that is needed) directly